angular-cli.json

```diff
@@ -21,8 +21,8 @@
         "styles.css"
       ],
       "scripts": [],
```

Comment:
i have no idea why it is added i had some trouble in deploying i followed the steps only , i don't know angular . Sorry no idea . should i delete it ?